sources/presentation/Stride.Core.Presentation/Themes/ThemeSelector.xaml

```diff
@@ -1801,6 +1802,122 @@
+            <Label Content=" d " Margin="2" HorizontalAlignment="Center" VerticalAlignment="Center" Grid.Column="1"/>
+            <ctrl:NumericTextBox Value="{Binding Hours, RelativeSource={RelativeSource Mode=TemplatedParent}, Converter={cvt:ToDouble}}" Grid.Column="2"
+                               DecimalPlaces="0" DisplayUpDownButtons="False" SelectAllOnFocus="True" Margin="-1" TextAlignment="Right" Minimum="0" Maximum="23"
+                               WatermarkContent="{Binding WatermarkContent, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                               WatermarkContentTemplate="{Binding WatermarkContentTemplate, RelativeSource={RelativeSource Mode=TemplatedParent}}"/>
+            <Label Content=" h " Margin="2" HorizontalAlignment="Center" VerticalAlignment="Center" Grid.Column="3"/>
+            <ctrl:NumericTextBox Value="{Binding Minutes, RelativeSource={RelativeSource Mode=TemplatedParent}, Converter={cvt:ToDouble}}" Grid.Column="4"
+                               DecimalPlaces="0" DisplayUpDownButtons="False" SelectAllOnFocus="True" Margin="-1" TextAlignment="Right" Minimum="0" Maximum="59"
+                               WatermarkContent="{Binding WatermarkContent, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                               WatermarkContentTemplate="{Binding WatermarkContentTemplate, RelativeSource={RelativeSource Mode=TemplatedParent}}"/>
+            <Label Content=" min " Margin="2" HorizontalAlignment="Center" VerticalAlignment="Center" Grid.Column="5"/>
+            <ctrl:NumericTextBox Value="{Binding Seconds, RelativeSource={RelativeSource Mode=TemplatedParent}}" Grid.Column="6"
+                               DecimalPlaces="3" DisplayUpDownButtons="False" SelectAllOnFocus="True" Margin="-1" TextAlignment="Right" Minimum="0.000" Maximum="59.999"
+                               WatermarkContent="{Binding WatermarkContent, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                               WatermarkContentTemplate="{Binding WatermarkContentTemplate, RelativeSource={RelativeSource Mode=TemplatedParent}}"/>
+            <Label Content=" s " Margin="2" HorizontalAlignment="Center" VerticalAlignment="Center" Grid.Column="7"/>
+          </Grid>
+        </ControlTemplate>
+      </Setter.Value>
+    </Setter>
+  </Style>
+
+  <Style TargetType="ctrl:DateTimeEditor">
+    <Setter Property="Focusable" Value="False"/>
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="ctrl:DateTimeEditor">
+          <Grid>
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition Width="*"/>
+              <ColumnDefinition Width="Auto"/>
+            </Grid.ColumnDefinitions>
+            <ctrl:TextBox Grid.Column="0" IsEnabled="False" SelectAllOnFocus="True" Margin="-1" TextAlignment="Left"
```

Comment:
Conside using `IsReadOnly` instead of `IsEnabled` so that the text isn't grayed out.

Reply:
Thanks for the hint! I amended the last commit.